### DISCUSSION TITLE
Maintenance, issues and operational Notices

### DISCUSSION THREAD

**Discussion by brackend** [argoproj/argo-cd#7101] [Q&A]
Hi, 

recently issue: https://github.com/argoproj/argo-cd/issues/7075 caused some delays for our developments teams.  They simply weren't aware of the issue and pursued several avenues of investigation before the issue was communicated to them.  

Of course, this can be addressed in the ways deployments are communicated.  But one way that might be useful is if there were some kind of notice on the home page of Argocd UI - for such issues as 7075 but also items such as upcoming maintenance.  
Question:  is there a way to include notices within the argocd ui.  Say for example on home page,  the appliactions tab,or even in the documentations tab, or in the header f all pages.

**Comment by chetan-rns** [comment]
Hi @brackend,
 
Argo CD supports the banner feature that can be used to notify users. This can be configured by setting the `ui.bannercontent` / `ui.bannerurl` in the `argocd-cm` configmap   https://github.com/argoproj/argo-cd/blob/master/CHANGELOG.md#banner-feature

**Comment by brackend** [comment]
thank you perfect.

**Comment by brackend** [comment]
It would be cool to have banners within the tiles for individual applications as well.

**Reply by todaywasawesome** [reply]
I suggest [creating an issue](https://github.com/argoproj/argo-cd/issues/new/choose), that's a cool feature idea.
